index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="./src/styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>A webpage about me</title>
+</head>
+<body>
+    <header>
+        <h1>Everything You Need to Know About Alex</h1>
+
+        <img id="image" src="https://d2g8igdw686xgo.cloudfront.net/37131652_1550357560852911_r.jpeg">
```

Comment:
Make sure to always include a descriptive alt attribute on img tags for accessibility and for those times when images won't load like slow connections, etc.

Reply:
Thank you, I totally forgot!